kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue

```diff
@@ -0,0 +1,483 @@
+        <dropdown-menu
+          :name="$tr('options')"
+          :options="lessonOptions"
+          @select="handleSelectOption"
+        />
+      </div>
+    </div>
+    <div>
+      <dl>
+        <dt>
+          {{ $tr('status') }}
+          <info-icon
+            :iconAriaLabel="$tr('lessonStatusDescription')"
+            :tooltipText="$tr('statusTooltipText')"
+            tooltipPosition="bottom left"
+          />
+        </dt>
+        <dd>
+          <status-icon :active="lessonActive" />
+          <k-button
+            appearance="basic-link"
+            class="change-status-button"
+            :text="$tr('changeLessonStatus')"
+            @click="currentAction=LessonActions.CHANGE_STATUS"
+          />
+        </dd>
+
+        <dt>
+          {{ $tr('description') }}
+        </dt>
+        <dd>
+          {{ lessonDescription || $tr('noDescription') }}
+        </dd>
+
+        <dt>
+          {{ $tr('visibleTo') }}
+        </dt>
+        <dd>
+          <template v-if="!lessonAssignments.length">
+            {{ this.$tr('noOne') }}
+          </template>
+          <template v-else-if="lessonIsAssignedToClass(lessonAssignments)">
+            {{ this.$tr('entireClass') }}
+          </template>
+          <ul
+            v-else
+            class="group-list"
+          >
+            <li
+              class="group-list-item"
+              v-for="assignment in lessonAssignments"
+              :key="assignment.id"
+            >
+              <span>{{ getGroupName(assignment) }}</span>
+            </li>
+          </ul>
+        </dd>
+      </dl>
+
+      <div class="resource-list">
+        <div class="resource-list-header">
+          <div class="resource-list-header-title-block">
+            <h2 class="resource-list-header-title">{{ $tr('resources') }}</h2>
+          </div>
+          <div class="resource-list-header-add-resource-button">
+            <k-router-link
+              :to="lessonSelectionRootPage"
+              :text="$tr('addResourcesButtonPrompt')"
+              :primary="true"
+              appearance="raised-button"
+            />
+          </div>
+        </div>
+      </div>
+
+      <core-table>
+        <thead slot="thead">
+          <tr>
+            <th class="visuallyhidden core-table-icon-col">
+              {{ $tr('resourceReorderColumnHeaderForTable') }}
+            </th>
+            <th class="core-table-main-col">
+              {{ $tr('nameColumnHeaderForTable') }}
+            </th>
+            <th>
+              {{ $tr('resourceProgressColumnHeaderForTable') }}
+            </th>
+            <th class="visuallyhidden">
+              {{ $tr('resourceRemovalColumnHeaderForTable') }}
+            </th>
+          </tr>
+        </thead>
+        <transition-group
+          name="resource-reorder"
+          slot="tbody"
+          class="core-table-rows-selectable"
+          tag="tbody"
+        >
+          <tr
+            v-if="!removals.includes(resourceId)"
```

Comment:
have a `removals` array storing the temp state of deleted resources (you can undo)